server/server.js

```diff
@@ -166,7 +192,12 @@ function findByUuid(req, res, uuid) {
 // API
 api.get("/documents/:uuid", jwtCheck, (req, res) => {
   const uuid = req.params.uuid;
-  findByPath(req, res, uuid);
+  const { version_date } = req.query;
+  if (!version_date) {
+    findByPath(req, res, uuid);
+  } else {
+    findByPathForVersionDate(req, res, uuid, version_date);
+  }
```

Comment:
Let's keep var names in camel case :)
```suggestion
  const { version_date: versionDate } = req.query;
  if (!versionDate) {
    findByPath(req, res, uuid);
  } else {
    findByPathForVersionDate(req, res, uuid, versionDate);
  }
```